### PR TITLE
Migration to gitHub action

### PR DESCRIPTION
**Pull request by ahalberkamp**
Remove travis and apply Github action for build.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,3 +230,58 @@ Additionally, response can contain additional response body. All fields of the r
...
+       ```
+
+## Build and deploy
+
+- This module is deployed as docker image to dockerhub.
+
+- The build and deployment of the docker are done using github actions.
+
+- On each push to the remote github repository, the github action [ci](https://github.com/commercetools/commercetools-paypal-plus-integration/actions/workflows/ci.yml) is triggered, which builds the project and
+ executes it`s tests.
```
Comment (lojzatran) [inline README.md:275]:
```suggestion
 executes its tests. 
```

**Review by lojzatran — CHANGES_REQUESTED** [review]

File: src/test/integration/java/com/commercetools/payment/handler/CommercetoolsGetPaymentsControllerIT.java [hunk 2]
```diff
@@ -46,6 +40,7 @@ public void tearDownAfterAll() {
     @Before
     @Override
     public void setUp() {
+        super.setupBeforeAll();
```
[the remark this answers is not in the document]
Comment (ahalberkamp) [reply src/test/integration/java/com/commercetools/payment/handler/CommercetoolsGetPaymentsControllerIT.java:43]: without this change the test was failing in the git-action

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```
[the remark this answers is not in the document]
Comment (ahalberkamp) [reply package-lock.json]: 9b1ace6

File: src/test/integration/java/com/commercetools/payment/handler/CommercetoolsGetPaymentsControllerIT.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (lojzatran) [reply src/test/integration/java/com/commercetools/payment/handler/CommercetoolsGetPaymentsControllerIT.java:43]: That's weird that changing from travis to github actions you have to change the tests too, but ok.

**Review by lojzatran — APPROVED** [review]

**Pull request merged**